lib/node-express/src/middleware/auth.ts

```
@@ -0,0 +1,35 @@
+
+/**
+ * Middleware that adds user information from headers to the request object.
+ * Expects x-user-id and x-user-email headers to be set by authentication layer.
+ * Throws 401 if headers are missing.
+ */
+export const auth: Handler = (req, res, next): void => {
+  const userId = req.headers["x-user-id"];
+  const userEmail = req.headers["x-user-email"];
+
+  if (!userId || !userEmail) {
+    return next(createError(401, "Unauthorized"));
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)